src/main/java/core/basesyntax/service/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static final String HEADER = "fruit,quantity";
+
+    @Override
+    public void writeToFile(String report, String fileName) {
+        File fruitReport = new File(fileName);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fruitReport))) {
+            writer.write(HEADER);
+            for (String line : report.split(";")) {
+                writer.newLine();
+                writer.write(line);
+            }
+            writer.flush();
+        } catch (IOException e) {
+            throw new RuntimeException(e);

```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/156338484/266724cf-e981-401c-afdf-db8c9981a0b1)
Try to use this case, it look better

Reply:
In this service you should only write prepared report